tensorflow/python/keras/activations.py

```diff
@@ -439,9 +440,18 @@ def deserialize(name, custom_objects=None):
       ValueError: `Unknown activation function` if the input string does not
       denote any defined Tensorflow activation function.
   """
+  globs = globals()
+
+  # only replace missing activations, when there are no custom objects
+  if custom_objects is None:
```

Comment:
This if statement can be removed -- [the custom object dict is always checked before the module objects](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/utils/generic_utils.py#L200), so it's safe to add the missing activations here.